src/vellum/workflows/nodes/displayable/inline_prompt_node/node.py

```diff
@@ -10,6 +11,15 @@
 from vellum.workflows.types.generics import StateType
 
 
+def strip_json_code_fences(text: str) -> str:
+    """Strip JSON code fences from text if present."""
+    pattern = r"^```(?:json)?\s*([\\s\\S]*?)```\s*$"
```

Comment:
Let's remove this logic for now since it's a bit more controversial whether we'd like to include